src/main/io/osd.c

```diff
@@ -143,7 +148,27 @@ typedef struct statistic_s {
     uint8_t min_link_quality;
 } statistic_t;
 
+typedef struct radioControls_s {
+    uint8_t left_vertical;
+    uint8_t left_horizontal;
+    uint8_t right_vertical;
+    uint8_t right_horizontal;
+} radioControls_t;
+
+typedef enum radioModes_e {
+    MODE1,
+    MODE2,
+    MODE3,
+    MODE4
+} radioModes_t;
+
 static statistic_t stats;
+static const radioControls_t radioModes[4] = {
```

Comment:
This needs to be made conditional, or the build will fail with a warning for targets without `USE_OSD_PROFILES`. (Protip: Having a look at the result of the CI checks will reveal this type of problem.)

Reply:
Thanks, yes I should probably use the CI checks and reap the benefits.